src/components/sidebar/routesView/routeShow.scss

```diff
@@ -56,6 +56,12 @@
                 font-size: $smallFontSize;
                 padding-left: 10px;
                 padding-top: 2px;
+                display: flex;
+
+                div:nth-child(2) {
+                    min-width: 20px;
+                    text-align: center;
+                }
```

Comment:
Would using a className like ".separator" plus padding left 10px, padding right 10px be more readable? I think so. Now when just watching this css, it is hard to see what this is used for

Reply:
Changed this to use class instead of pseudoclass